packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -97,8 +97,13 @@ contract Bridge is EssentialContract, IBridge {
             revert B_INVALID_CHAINID();
         }
 
-        if (message.to == address(0)) revert B_INVALID_TO();
-        if (message.to == destBridge) revert B_INVALID_TO();
+        if (
+            message.to == address(0) || message.to == destBridge
```

Comment:
@Brechtpd Here, maybe I can simply remove this if-statement.

Reply:
Ah yeah, that makes sense!